src/ray/common/ray_config_def.h

```diff
@@ -330,9 +330,17 @@ RAY_CONFIG(int64_t, max_placement_group_load_report_size, 100)
 /// Python IO workers to determine how to store/restore an object to/from
 /// external storage.
 RAY_CONFIG(std::string, object_spilling_config, "")
+
 /// Whether to enable automatic object spilling. If enabled, then
 /// Ray will choose objects to spill when the object store is out of
 /// memory.
 RAY_CONFIG(bool, automatic_object_spilling_enabled, true)
+
 /// The maximum number of I/O worker that raylet starts.
 RAY_CONFIG(int, max_io_workers, 1)
+
+/// The minimum size that can be spilled. 100 MB by default.
+/// This value is recommended to be smaller than the plasma object size, but
+/// it will still work although it doesn't.
```

Comment:
I don't understand this comment. What is the "plasma object size"? And why is this the recommendation? It would be good to clarify what would happen if this condition is not met.

Reply:
My bad. I mean `--plasma-store-size`. If the size is bigger than `--plasma-store-size`, we will spill every pinned object every time. Maybe we should strictly disallow that.

Reply:
Or we should probably auto calculate this value (like at maximum 60% of plasma store size or something)